mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -279,11 +279,19 @@ class JsonSerialize(mailboxIdFactory: MailboxId.Factory, messageIdFactory: Messa
     case _ => JsError()
   }
   implicit val messageUidReads: Reads[MessageUid] = {
-    case JsNumber(value) => JsSuccess(MessageUid.of(value.toLong))
+    case JsNumber(value) => if (value.isValidLong) {
+        JsSuccess(MessageUid.of(value.toLong))
+      } else {
```

Comment:
the else is useless

Reply:
See https://github.com/linagora/james-project/pull/2927